Add additional info for integrations using polling

Alex9779 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->

## Proposed change
<!-- 
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request.
-->
Add additional info for integrations using polling
exposing SCAN_INTERVAL as option is no longer accepted

## Type of change
<!--
  What type of change does your pull request introduce to Home Assistant Developer Documentation? Put an `x` in the appropriate box
  NOTE: Please, check only 1! box! 
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [x] Document existing features within Home Assistant
- [ ] Document new or changing features which there is an existing pull request elsewhere
- [ ] Spelling or grammatical corrections, or rewording for improved clarity
- [ ] Changes to the backend of this documentation
- [ ] Removed stale or deprecated documentation

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
  
  For documentation relating to existing code please link to the relevant file on the appropriate master or dev branch (i.e.: https://github.com/home-assistant/core/blob/7c784b69638f3e2b3c91294b31a62e1058ba9709/homeassistant/components/random/sensor.py#L48-L57)

  For documentation relating to new or changing code, please link to the corresponding pull request (i.e. home-assistant/core#2). This lets us easily check the status of your proposal.
-->

- This PR fixes or closes issue: fixes #
- Link to relevant existing code or pull request:

MartinHjelmare commented on docs/integration_fetching_data.md:
````diff
@@ -149,6 +149,10 @@ from datetime import timedelta
 SCAN_INTERVAL = timedelta(seconds=5)
 ```
 
+:::info
````
I think we should put this info under a separate header so we can easily link to it. The current header where this note is placed is also not correct since this is not only for separate polling for each individual entity.

Alex9779 replied:
Accepted but then the last paragraph with the code block is also not for individual entities.
To structure it I would introduce a section between "Coordinated ..." and "Separate ..." maybe "Setting a polling interval"? And move the last paragraph of "Separate .." and the code block in there along with the note?

MartinHjelmare replied:
The last paragraph is correct. It's only for individually polled entities.

MartinHjelmare reviewed (changes requested):
See above.

Alex9779 commented:
Hmmm and BTW maybe `SCAN_INTERVAL` should be in the large code block above, there the interval is set directly and now variable is used, and why is it not `CONF_SCAN_INTERVAL` imported from homeassistant.const or do we not care about using that?

Alex9779 commented:
Ok sorry I am out, that's what I expected, I stumbled into a problem, tried to fix it, learned something that you expect it a different way but even with about 1,5 days of effort I wasn't able to find all information and you want me then to correct that. Yeah I would like to help out but obviously I don't get the whole meaning of what is written on that page.
So I don't think I will be able to provide an acceptable change... Sorry...

MartinHjelmare commented:
To be clear I only asked for a separate new header for the new information. But I understand if you don't want to continue with this. Thanks for your effort. 👍

Alex9779 commented:
All good, tried it, failed it... Thought I could do it to help some others reading that to prevent them to do something like me which is not state of the art...
But honestly docs like this should be written by someone seeing the whole picture, that is not me...

Alex9779 commented:
@MartinHjelmare I am sorry to bug you this way but this is frustrating, I removed my changes to the integration reducing the polling interval and tried the "recommended way". Due to the lack of documentation also on the user side of the documentation this didn't work for about a day because I added an automation for update_entity for the device which has six entities. But this does not work.
I have to update an entity with the service.
As I am aware of what goes on in the background I didn't set it to update all entities which would query the cloud service six times which would work but not consistently because that cloud service has a rate limit of 5 queries per second and 30 queries per hour.
I am unsure about what to do and where to post to get that fixed or added also to normal user docs so that other users do not face the same problems as I do...

MartinHjelmare commented:
As every integration can have different entity vs api setups with different rate limits I suggest this documentation belongs in each integration docs if needed. Here's an existing example:
https://www.home-assistant.io/integrations/google_travel_time/#updating-sensors-on-demand-using-automation
